content/en/synthetics/private_locations/_index.md

```diff
@@ -49,9 +45,33 @@ Your private location worker pulls your test configurations from Datadog's serve
 
 In order to use private locations for [Continuous Testing tests][23], you need v1.27.0 or later.
 
-### Docker
+{{< tabs >}}
+{{% tab "Docker" %}}
+
+Private locations are Docker containers that you can install anywhere inside your private network. You can access the [private location worker image][101] on Google Container Registry. It can run on a Linux-based OS or Windows OS if the [Docker engine][102] is available on your host and can run in Linux containers mode.
+
+[101]: https://console.cloud.google.com/gcr/images/datadoghq/GLOBAL/synthetics-private-location-worker?pli=1
+[102]: https://docs.docker.com/engine/install/
+
+{{% /tab %}}
+{{% tab "Windows" %}}
+
+Private locations are Windows services that you can install anywhere inside your private network using an [MSI file][101]. Run this file from the virtual or physical machine that you would like to install the private location on, and in a cloud provider VM matching the following requirements. Datadog recommends running the private location worker on a virtual machine or Windows container.
```

Comment:
I'm confused by the second sentence, particularly the word "and." Would it make sense to just say "Run this file from the virtual or physical machine that you would like to install the private location on. This machine's requirements are listed in the table below."?

Reply:
```suggestion
Private locations are Windows services that you can install anywhere inside your private network using an [MSI file][101]. Run this file from the virtual or physical machine that you would like to install the private location on. This machine's requirements are listed in the table below. Datadog recommends running the private location worker on a virtual machine or Windows container.
```